src/Fractal.php

```diff
@@ -21,7 +22,7 @@ class Fractal implements JsonSerializable
     /** @var string|\League\Fractal\Serializer\SerializerAbstract */
     protected $serializer;
 
-    /** @var string|callable|\League\Fractal\TransformerAbstract */
+    /** @var string|callable|\League\Fractal\TransformerAbstract|null */
```

Comment:
Just update the outdated docblock, because the transformer can be `null` already.